tests/views/playingScreen-test.js

```diff
@@ -37,16 +38,62 @@ describe('PlayingScreen', function () {
     TestUtils.Simulate.mouseUp(screen[0]);
     expect(clicked).toBe(true);
   });
+  it('creates a PlayingScreen and checks mouseDown, mouseUp with video360', function() {
+    var isTouched = false
+      , isStartHideControlBarTimer = false;
```

Comment:
can you use var on the second line as well?  Otherwise you can technically just do 

var isTouched = false
      , isStartHideControlBarTimer = false
      , mockController = {

And the current team tries to avoid multiple variable definitions using ','